src/Components/Navigation/Navigation.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Navigation.css';
+
+const Navigation = () => (
+  <nav>
+    <NavLink activeClassName="active" to="/add-view">
+      Add Item
+    </NavLink>
+    <NavLink activeClassName="active" to="/list-view">
+      View List
+    </NavLink>
```

Comment:
Create component structure. I like the usage of  `NavLink` here (instead e.g. `Link`) 🔝

Reply:
Thank you for the recommendations @YuriDevAT. We'll work on them stat!